examples/everything.amp.html

```diff
@@ -314,7 +314,7 @@ <h1 id="top">AMP #0</h1>
   <amp-instagram
     data-shortcode="fBwFP"
     width="320"
-    height="392"
+    height="320"
```

Comment:
As far as I can tell the correct ratio is squared for these examples. So not sure why they weren't squared before. Is it on purpose? the 392 leaves a big padding at the bottom that the image placeholder slips into and hence stretches and no longer matches the iframe.